<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnOrganizationDialog</class>
 <widget class="QDialog" name="ColumnOrganizationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Column organization dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="c1Box">
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>URL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Login</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Password</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="c2Box">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>URL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Login</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Password</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="c3Box">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>URL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Login</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Password</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose custom column organization</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="c4Box">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>URL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Login</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Password</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notes</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColumnOrganizationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColumnOrganizationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
